### PR TITLE
Add FriendNotifier

### PR DESCRIPTION
**Pull request by isaquall**
this is my first extension, so let me know if anything needs fixing. thanks! :)

### CODE REVIEW

**Comment by WiredSpast** [comment]
Code wise, I have one suggestion:
https://github.com/isaquall/FriendNotifier/blob/main/src/main/java/me/isaquall/friendnotifier/FriendNotifier.java#L118-L146

You don't block the command being sent, might be better if you did, when using a switch there are 2 ways to do this:
1. Add `hMessage.setBlocked(true)` to each case block
2. Put `hMessage.setBlocked(true)` before the switch statement and put `hMessage.setBlocked(false)` in a default block
    ```java
    hMessage.setBlocked(true);
    switch(message) {
        case "...": ...
        case "...": ...
        default:
            hMessage.setBlocked(false);
            break;
    }
    ```

File: store/extensions/FriendNotifier/extension.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  "description": "Know when your friends log on and off, and more!\"\n\n:playerlist, :pl - Displays Habbos in current room.\n:mutefriendjoins, :mf - Mutes notifications on friend join/leave. Send again to toggle on/off.\n:muteroomjoins, :mr - Mutes notifications when Habbos join/leave the current room. Send again to toggle on/off.",
+  "authors": [
+    {
+      "name": "isaquall",
+      "hotel": ".com",
+      "username": "isaquall",
+      "discord": "isaquall"
+    }
+  ],
+  "version": "1.0.0",
```
Comment (WiredSpast) [inline store/extensions/FriendNotifier/extension.json]: The version listed here doesn't match with the version listed in the extensions code https://github.com/isaquall/FriendNotifier/blob/main/src/main/java/me/isaquall/friendnotifier/FriendNotifier.java#L19

**Comment by isaquall** [comment]
If I'm not misunderstanding, then I already use the first option you mentioned. Packets are blocked from being sent to the server here:
- https://github.com/isaquall/FriendNotifier/blob/329908046bc98e4739ca83a3bad113c51c95d0cf/src/main/java/me/isaquall/friendnotifier/FriendNotifier.java#L123
- https://github.com/isaquall/FriendNotifier/blob/329908046bc98e4739ca83a3bad113c51c95d0cf/src/main/java/me/isaquall/friendnotifier/FriendNotifier.java#L128
- https://github.com/isaquall/FriendNotifier/blob/329908046bc98e4739ca83a3bad113c51c95d0cf/src/main/java/me/isaquall/friendnotifier/FriendNotifier.java#L138

**Comment by WiredSpast** [comment]
> If I'm not misunderstanding, then I already use the first option you mentioned. Packets are blocked from being sent to the server here:
> 
> * https://github.com/isaquall/FriendNotifier/blob/329908046bc98e4739ca83a3bad113c51c95d0cf/src/main/java/me/isaquall/friendnotifier/FriendNotifier.java#L123
> * https://github.com/isaquall/FriendNotifier/blob/329908046bc98e4739ca83a3bad113c51c95d0cf/src/main/java/me/isaquall/friendnotifier/FriendNotifier.java#L128
> * https://github.com/isaquall/FriendNotifier/blob/329908046bc98e4739ca83a3bad113c51c95d0cf/src/main/java/me/isaquall/friendnotifier/FriendNotifier.java#L138

Ow lol my bad, I was sleepy when I looked at it so I must have somehow looked over it 😬, in that case the only thing incorrect is the mismatching of the version number, for which I apparently hadn't clicked on send review yet until now...

**Comment by isaquall** [comment]
> Ow lol my bad, I was sleepy when I looked at it so I must have somehow looked over it 😬, in that case the only thing incorrect is the mismatching of the version number, for which I apparently hadn't clicked on send review yet until now...

haha you're good! I've also changed the version now too.

**Pull request merged**